README.md

```diff
@@ -9,3 +9,8 @@
 - `git push -u origin main` / `git push`
 - `git remote -v`
 - `git remote add origin <ssh link>`
```

Comment:
We could use a html link

Reply:
True but I leave it like this cause we always want to use the ssh link.